### DISCUSSION TITLE
Are simple-openai tests flaky?

### DISCUSSION THREAD

**Discussion by the-gigi** [sashirestela/simple-openai#209] [Q&A] [answered]
We recently forked **simple-openai** due these deserialization issues:
https://github.com/sashirestela/cleverclient/issues/77
https://github.com/sashirestela/simple-openai/issues/205

The only change is that instead of using **Cleverclient**'s `jsonToObject()` we provide a local implementation with an ObjectMapper that can handle dates

We noticed that many of the tests fail intermittently. This happens mostly in our CI/CD system (Github Actions). I don't remember such intermittent failures when I was running the tests locally. In the interest of minimizing noise we disabled all the tests for now.

Is that a known phenomenon?

**Comment by sashirestela** [comment, marked as answer]
Hi @the-gigi. No, it is not. Tests should not fail.
